### ISSUE TITLE
Relation to Glove and word2vec

### ISSUE THREAD

**Issue by alexanderpanchenko** [uhh-lt/path2vec#26]
Add a description (in one paragraph) which relates our model (in terms of the math notation already introduced in the paper) to the Glove https://nlp.stanford.edu/pubs/glove.pdf and word2vec.  

Glove:

<img width="495" alt="image" src="https://user-images.githubusercontent.com/1456830/49151850-663bf880-f311-11e8-9343-e3950e5825d6.png">

<img width="492" alt="image" src="https://user-images.githubusercontent.com/1456830/49151838-60461780-f311-11e8-8141-fd0bbb8e905e.png">

**Comment by alexanderpanchenko** [comment]
rewrite Glove and SkipGram objectives here in our notation + add a few sentences about this.

**Comment by alexanderpanchenko** [comment]
<img width="869" alt="image" src="https://user-images.githubusercontent.com/1456830/49586243-ea6d2c00-f960-11e8-87b4-10c0214019b0.png">


<img width="855" alt="image" src="https://user-images.githubusercontent.com/1456830/49586248-ef31e000-f960-11e8-9cd0-0973308d87fc.png">

<img width="855" alt="image" src="https://user-images.githubusercontent.com/1456830/49586260-f6f18480-f960-11e8-90ad-9fc4e69327b6.png">

**Comment by alexanderpanchenko** [comment]
@m-dorgham any updates on this?

**Comment by m-dorgham** [comment]
@alexanderpanchenko Have a look at this paragraph: https://www.overleaf.com/9629192926rfsnkrbwyyyt

Is this enough or do I need to explain more?

**Comment by alexanderpanchenko** [comment]
In the last equation instead of exp should be sigma??

**Comment by m-dorgham** [comment]
The equation with the sigma is the final objective function after introducing the concept of negative sampling. The one I wrote is the general objective function (this is how it was written in the GloVe paper as well). 
We can write it with the sigma notation or leave it in the general form, as you like. But if we wrote it in the sigma notation we might need to explain in more words the concept of negative sampling.
